packages/eslint-plugin-react-hooks/__tests__/ESLintRuleExhaustiveDeps-test.js

```diff
@@ -7345,6 +7860,24 @@ const testsTypescript = {
         },
       ],
     },
+    {
+      code: normalizeIndent`
+        function Foo() {
+          const foo = {} as any;;
```

Comment:
Nit: double ;

Reply:
Fixed in https://github.com/facebook/react/pull/19590/commits/eaa2aff9424e7ded164799fa2aef27ccfa4ed3d0